.travis.yml

```diff
@@ -75,19 +80,20 @@ install:
      $HOME/miniconda/bin/conda info -a;
      $HOME/miniconda/bin/conda create -q -n test-environment python=$PYTHON_VERSION"
 
- # Activate conda environment ad install required packages
+  # Activate conda environment ad install required packages
   - docker exec -t ubuntu-test bash -c "
       source $HOME/miniconda/bin/activate test-environment;
       pip install --user -r $HOME/spark-deep-learning/dev/dev-requirements.txt;
       pip install --user -r $HOME/spark-deep-learning/python/requirements.txt;"
 
-script:
   - docker cp $HOME/.cache ubuntu-test:$HOME/
   # build assembly
   - docker exec -t ubuntu-test bash -c "
       source $HOME/miniconda/bin/activate test-environment;
       cd $HOME/spark-deep-learning;
       ./dev/run.py assembly"
+
```

Comment:
I moved this here since this folds the above step in travis ci, saves us a lot of time scrolling. i do not feel strongly about this tho. cc @MrBago